specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/examples/ApiManagementListGatewaysBySubscription.json

```
@@ -64,6 +64,37 @@
               "name": "WorkspaceGatewayPremium",
               "capacity": 1
             }
+          },
+          {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateways/standard-gw-2",
+            "name": "standard-gw-2",
+            "type": "Microsoft.ApiManagement/gateways",
+            "tags": {
+              "Owner": "vitaliik"
+            },
+            "location": "East US",
+            "etag": "AAAAAAAWKwo=",
+            "properties": {
+              "provisioningState": "Succeeded",
+              "targetProvisioningState": "",
+              "createdAtUtc": "2021-06-16T09:40:00.9453556Z",
+              "frontend": {
+                "defaultHostname": "standard-gw-2.eastus.gateway.azure-api.net"
+              },
+              "backend": {
+                "subnet": {
+                  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/vn2/subnets/sn2"
+                }
+              },
+              "configurationApi": {
+                "hostname": "standard-gw-2.configuration.eastus-01.azure-api.net"
+              }
+            },
```

Updating name and removing config API
```suggestion
            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateways/kube-managed-gw-1",
            "name": "kube-managed-gw-1",
            "type": "Microsoft.ApiManagement/gateways",
            "tags": {
              "Owner": "vitaliik"
            },
            "location": "East US",
            "etag": "AAAAAAAWKwo=",
            "properties": {
              "provisioningState": "Succeeded",
              "targetProvisioningState": "",
              "createdAtUtc": "2021-06-16T09:40:00.9453556Z",
              "frontend": {
                "defaultHostname": "kube-managed-gw-1.eastus.gateway.azure-api.net"
              },
              "backend": {
                "subnet": {
                  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/vn2/subnets/sn2"
                }
              }
            },
```

---

Done.